finder.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+return \StubsGenerator\Finder::create()
+    ->in('source/vendor/wp-cli/wp-cli')
+    ->append(\StubsGenerator\Finder::create()->files()->depth('< 4')->in(['source/vendor/wp-cli/wp-cli/php']))
```

Comment:
these "append" are not doing anything

Reply:
Basically we need two things from a command or wp-cli/wp-cli:

- the main command file
- and classes

Sometimes the command file is almost empty: https://github.com/wp-cli/core-command/blob/master/core-command.php so only the classes

Reply:
Actually all the `*command.php` files are tiny: WP-CLI must have moved everything into classes.

Reply:
changed the depth to 2